src/tensorlake/function_executor/proto/function_executor.proto

```diff
@@ -161,6 +173,128 @@ message InfoResponse {
     optional string sdk_language_version = 4;
 }
 
+// === run_task_allocations_session RPC === //
+
+message SerializedObjectID {
+    optional string id = 1;
+}
+
+message SerializedObjectManifest {
+    optional SerializedObjectID id = 1;
+    optional SerializedObjectEncoding encoding = 2;
+    optional uint64 encoding_version = 3;
+    optional uint64 size = 4;
+    optional string sha256_hash = 5;
+    optional uint32 chunk_count = 6;
```

Comment:
Do we need a chunk count?

Reply:
Yep, this allows to avoid sending "file transmission finished" message. Chunk size is not fixed in the protocol.

Reply:
I'll remove chunk count too, both sides can understand if they got all the data by comparing the size they have recieved.